test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -59,14 +59,14 @@ Dump-dot-merlin includes the melange flags
 
   $ dune ocaml dump-dot-merlin $PWD
   EXCLUDE_QUERY_DIR
-  STDLIB /MELC_STDLIB/runtime/melange
-  B /MELC_STDLIB/belt/melange
+  STDLIB /MELC_STDLIB/melange
+  B /MELC_STDLIB/melange
+  B /MELC_STDLIB/melange
```

Comment:
@anmonteiro do you know why this kind of change happens? I think we should still tell merlin about the paths to belt and runtime libs, right?

I am testing melange from commit [cd739fe](https://github.com/melange-re/melange/commit/cd739fe78b5818c50c6438b76f02b6402deb3d4c) and dune from commit 7bb6de7087e1a0a50e8cc9836d2a0e8bc200fc25 and when I run `dune ocaml dump-dot-merlin .` I still see the original paths to runtime and belt, e.g.

```
STDLIB /home/me/code/monorepo/_opam/lib/melange/melange
B /home/me/code/monorepo/_opam/lib/melange/belt/melange
B /home/me/code/monorepo/_opam/lib/melange/melange
B /home/me/code/monorepo/_opam/lib/melange/runtime/melange
```

Reply:
opened https://github.com/ocaml/dune/pull/7622 (which I'll close briefly) to showcase how not mapping the prefixes changes the test.

What changed was the order of `melc --where` paths, nothing to worry about.